de_de.lang

```diff
@@ -249,4 +249,9 @@ gui.xaero_yaw=Richtung
 gui.xaero_yellow=Gelb
 gui.xaero_zoom=Zoom
 gui.xaero_zoom_in=Minikarte hereinzoomen
-gui.xaero_zoom_out=Minikarte herauszoomen
```

Comment:
Mistake Sorry

Reply:
Not a mistake, the difference is the "new line" symbol at the end, so it's all good!